UI/src/view/linear_asset/laneModellingForm.js

```diff
@@ -0,0 +1,597 @@
+        return formFields['sidecode_'+sideCode];
+      };
+
+      this.getAllFields = function(){
+        return _.flatten(_.map(formFields, function(form) {
+          return form;
+        }));
+      };
+
+      this.removeFields = function(sideCode){
+        if(!formFields['sidecode_'+sideCode])
+          throw Error("The form of the sidecode " + sideCode + " doesn't exist");
+
+        formFields['sidecode_'+sideCode] = [];
+      };
+    };
+
+    var forms = new AvailableForms();
+
+    self.initialize = function(assetTypeConfiguration, feedbackModel){
+      var rootElement = $('#feature-attributes');
+      self._assetTypeConfiguration = assetTypeConfiguration;
+      lanesAssets = self._assetTypeConfiguration.selectedLinearAsset;
+
+      eventbus.on('laneModellingForm: reload', function() {
+        reloadForm($('#feature-attributes'));
+      });
+
+      function setInitialForm() {
+        defaultFormStructure = formStructure;
+        currentFormStructure = mainLaneFormStructure;
+        isAddByRoadAddressActive = false;
+        lanesAssets.setCurrentLane(parseInt(_.min(_.map(self._assetTypeConfiguration.selectedLinearAsset.get(), function (lane) {
+          return _.head(_.find(lane.properties, function (property) {
+            return property.publicId == "lane_code";
+          }).values).value;
+        }))));
+      }
+
+      new FeedbackDataTool(feedbackModel, assetTypeConfiguration.layerName, assetTypeConfiguration.authorizationPolicy, assetTypeConfiguration.singleElementEventCategory);
+
+      eventbus.on(self.events('selected'), function () {
+        setInitialForm();
+        reloadForm(rootElement);
+      });
+
+      eventbus.on(self.events('unselect', 'cancelled'), function() {
+        rootElement.find('#feature-attributes-header').empty();
+        rootElement.find('#feature-attributes-form').empty();
+        rootElement.find('#feature-attributes-footer').empty();
+        dateutil.removeDatePickersFromDom();
+      });
+
+      eventbus.on('closeForm', function() {
+        rootElement.find('#feature-attributes-header').empty();
+        rootElement.find('#feature-attributes-form').empty();
+        rootElement.find('#feature-attributes-footer').empty();
+        dateutil.removeDatePickersFromDom();
+      });
+
+      eventbus.on('layer:selected', function(layer) {
+        if(self._assetTypeConfiguration.layerName === layer){
+          $('ul[class=information-content]').empty();
+
+          if(self._assetTypeConfiguration.isVerifiable)
+            self.renderLinkToWorkList(layer);
+          if(self._assetTypeConfiguration.hasInaccurate)
+            self.renderInaccurateWorkList(layer);
+        }
+      });
+
+      eventbus.on('application:readOnly', function(){
+        if(self._assetTypeConfiguration.layerName ===  applicationModel.getSelectedLayer() && self._assetTypeConfiguration.selectedLinearAsset.count() !== 0) {
```

Comment:
Check if it's possible to use:
```javascript
!_.isEmpty(self._assetTypeConfiguration.selectedLinearAsset)
```

instead:
```javascript
self._assetTypeConfiguration.selectedLinearAsset.count() !== 0

Reply:
Done

Reply:
Can be changed to this
```javascript 
!_.isEmpty(self._assetTypeConfiguration.selectedLinearAsset.get())
```